data/shipsystems/ship_systems.csv

```diff
@@ -25,8 +25,8 @@ Assume Wing Control,diableavionics_assumeControl,,,,,,,,,,0.1,30,0.1,15,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Botte Secrète,diableavionics_slash,,,,,,,,,,0.5,2,0.5,18,,,,,,,,,,,,,graphics/icons/hullsys/teleporter.png
 Hold Your Ground,diableavionics_dakka,,,,,,,,,,0.5,7,1,15,,,,,,,,,,,,,graphics/icons/hullsys/ammo_feeder.png
-Berserker,diableavionics_berserk,,,,,,,,,,0.5,6,1,15,,,,,,,,,,,,,graphics/icons/hullsys/ammo_feeder.png
-Aether Splitter,diableavionics_splitter,,,,,,,,,,1.5,1.25,1.5,20,,,,,TRUE,,,,,TRUE,,,graphics/icons/hullsys/high_energy_focus.png
+Berserker,diableavionics_berserk,,,,,,,,,,0.5,9,0.5,15,,,,,,,,,,,,,graphics/icons/hullsys/ammo_feeder.png
```

Comment:
avalanche system now lasts 3 seconds longer (50%!). keep in mind their changes also nerf the Energy RoF mult (snowblast shotgun) from 300% to 150%, but also add a Ballistic RoF mult of the same value. they also removed the movespeed reduction.